docs/exploration-and-organization/exploration.md

```diff
@@ -82,14 +82,14 @@ Searches take into account items’ titles, descriptions, and other metadata —
 
 **Bookmarks** are a way to quickly get back to things you visit frequently (or have been working on recently). Bookmarked items show up in the main navigation sidebar above [collections][collections].
 
-To bookmark an item, look for the **ribbon** icon. If you don't immediately see the ribbon, open up the settings for that item. You can bookmark:
+To bookmark an item, look for the **ribbon** icon in the upper right corner of the item's page. You can bookmark:
 
-- Questions (ribbon's in the editing sidebar)
-- Models (ribbon's in the editing sidebar)
-- Dashboards (ribbon's in the **...** menu)
-- Collections (ribbon's in the upper right of the collection's page)
+- Questions
+- Models
+- Dashboards
+- Collections
 
-When viewing a collection, you can also click on the **...** next to an item to add or remove a bookmark from the item.
+When viewing a collection, you can also click on the `...` next to an item to add or remove a bookmark from the item.
```

Comment:
Minor:

```suggestion
To add or remove a bookmark from an item, click on the three-dot menu (**...**) next to the item in the collection.
```
We've referred to this menu like so elsewhere, so might as well be consistent. Generally we bold UI elements people can click on.

And it's good to lead with what we're doing, then follow with how to do it, so people can stop reading if they don't care.

Reply:
Ah ok it was in `...` also elsewhere in the article. I liked it more in `...` because otherwise it looks like punctuation. But if that's consistent with the rest of the docs, then it's better to keep it bold.

Completely agree with leading with what we're doing!